packages/common/src/ui/components/inputs/Autocomplete/Autocomplete.tsx

```diff
@@ -79,7 +79,11 @@ export function Autocomplete<T>({
     <BasicTextInput
       {...props}
       autoFocus={autoFocus}
-      InputProps={{ disableUnderline: false, ...props.InputProps }}
+      InputProps={{
+        disableUnderline: false,
+        style: props.disabled ? { paddingLeft: 0 } : {},
```

Comment:
oh. yeah. the autocomplete used by the item selector wasn't left aligned for internal orders, so I tweaked.